test/anacondalib.py

```diff
@@ -72,7 +72,7 @@ def setUp(self):
         method = getattr(self, self._testMethodName)
         openqa_test = getattr(method, "openqa_test", "")
         wikictms_section = getattr(method, "wikictms_section", "")
-        boot_modes = getattr(method, "boot_modes", [])
+        boot_modes = getattr(method, "boot_modes", ["bios"])
```

Comment:
I guess this commit & the next one will go away once the test fixes are merged & this PR gets rebased ? :)

Reply:
Yes.